server.js

```diff
@@ -1,3 +1,6 @@
+if (process.env.NODE_DEBUG) {
```

Comment:
I made the same mistake in my suggestion

```suggestion
if (!process.env.NODE_DEBUG) {
```